src/main/telemetry/ibus_shared.c

```diff
@@ -33,19 +33,41 @@
 #include "telemetry/telemetry.h"
 #include "telemetry/ibus_shared.h"
 
-static uint16_t calculateChecksum(const uint8_t *ibusPacket, size_t packetLength);
-
+static uint16_t calculateChecksum(const uint8_t *ibusPacket);
 
 #if defined(TELEMETRY) && defined(TELEMETRY_IBUS)
-
+#if defined(GPS) && FLASH_SIZE > 256
+#define TELEMETRY_IBUS_EXTENDED
+#endif
+#include "config/feature.h"
 #include "config/parameter_group.h"
 #include "config/parameter_group_ids.h"
 #include "sensors/battery.h"
 #include "fc/rc_controls.h"
+#include "fc/config.h"
 #include "sensors/gyro.h"
+#include "drivers/accgyro/accgyro.h"
+#include "fc/runtime_config.h"
+#include "sensors/acceleration.h"
+#include "sensors/sensors.h"
+#include "sensors/barometer.h"
+#include "flight/imu.h"
+#include "flight/altitude.h"
+#include "flight/navigation.h"
+#include "io/gps.h"
 
 
 #define IBUS_TEMPERATURE_OFFSET  (400)
```

Comment:
No braces needed for a number.

Reply:
The code is inconsistent - there is number of places where braces are being used. Removed as requested.